### ISSUE TITLE
Fail to load infrastructure details

### ISSUE THREAD

**Issue by titirigaiulian** [spinnaker/spinnaker#6181]
### Issue Summary:

Fail to load infrastructure details for new deployed k8s manifests

### Cloud Provider(s):
Kubernetes

### Environment:
Spinnaker 1.23.1 is running in a kubernetes cluster backed by AWS EKS.

### Feature Area:
kubernetes, clouddriver, caching

### Steps to Reproduce:
1. Deploy a new k8s manifest containing a `Deployment`
2. Go to `Clusters`
3. Select the created `Pod`

- UI gets stuck in loading details
- Calls to `/manifests/{account}/{namespace}/pod%20{pod}` return 403 forbiden

#### Findings:

The issue is happening when it tries to sort the events [here](https://github.com/spinnaker/clouddriver/blob/6815f341437cf0a72f0442337801c39a976bf988/clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/provider/KubernetesManifestContainerBuilder.java#L52) when there are events with existing `lastTimestamp` key but with null value. 

### Additional Details:

Logs:
```
WARN 1 --- [0.0-7002-exec-3] c.n.s.c.k.c.ManifestController           : Failed to read manifest

java.lang.NullPointerException: null

WARN 1 --- [0.0-7002-exec-3] c.n.s.f.shared.FiatPermissionEvaluator   : Permission denied because at least one of the required arguments was null. resourceName=null, resourceType=APPLICATION, authorization=READ
```

**Comment by babinin** [comment]
same issue wtith 1.23.2

**Comment by ezimanyi** [comment]
Fixed in spinnaker/clouddriver#5099.
